.rubocop.yml

```diff
@@ -324,7 +324,7 @@ RSpec/MultipleMemoizedHelpers:
   Enabled: false
 RSpec/NestedGroups:
   Enabled: false
-Rspec/MessageSpies:
+RSpec/MessageSpies:
```

Comment:
rubocop was very mildly complaining about the casing here in its output, so I went ahead and fixed it.